x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/RuleQueryBuilder.java

```diff
@@ -75,7 +75,7 @@ public class RuleQueryBuilder extends AbstractQueryBuilder<RuleQueryBuilder> {
 
     @Override
     public TransportVersion getMinimalSupportedVersion() {
-        return TransportVersion.V_8_500_024;
+        return TransportVersion.V_8_500_031;
```

Comment:
Can you check that we don't need to bump the version if we add a new query?

Reply:
Also note that the latest version has changed, see https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/TransportVersion.java#L167

Reply:
This is required, bumping the version.